### PR TITLE
Fix bug: skip no exported field on readStructMetadata

### PR DESCRIPTION
**Pull request by 838239178**
Testing passed 100%

### CODE REVIEW

**Comment by 838239178** [comment]
Related issue: #105

File: example_test.go [hunk 1]
```diff
@@ -271,3 +272,21 @@ func ExampleUsage() {
...
+			Host string `yaml:"host" env:"SRV_HOST,HOST" env-description:"Server host" env-default:"localhost"`
+			Port string `yaml:"port" env:"SRV_PORT,PORT" env-description:"Server port" env-default:"8080"`
+		} `yaml:"server"`
+	}{}
+
+	if err := cleanenv.ReadConfig("example/simple_config/config.yml", &conf); err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("%+v", conf)
+}
```
Comment (ilyakaznacheev) [inline example_test.go:292]: Please add a newline at the file end.

File: example_test.go [hunk 2]
```diff
@@ -271,3 +272,21 @@ func ExampleUsage() {
...
+		server struct {
+			Host string `yaml:"host" env:"SRV_HOST,HOST" env-description:"Server host" env-default:"localhost"`
+			Port string `yaml:"port" env:"SRV_PORT,PORT" env-description:"Server port" env-default:"8080"`
+		} `yaml:"server"`
+	}{}
+
+	if err := cleanenv.ReadConfig("example/simple_config/config.yml", &conf); err != nil {
+		t.Fatal(err)
+	}
+	t.Logf("%+v", conf)
```
Comment (ilyakaznacheev) [inline example_test.go:291]: The example has to print some output and compare it with [a predefined text comment](https://go.dev/blog/examples). Either change it to do some output with the expected value or change the example to test (I prefer the latter). 

Also, it has to actually check something - like if exported fields are filled and not exported fields are empty.

**Review by ilyakaznacheev — CHANGES_REQUESTED** [review]

File: example_test.go (hunk 1 above)
Comment (838239178) [reply example_test.go]: I have a newline on my local editor but it seems not pushing to remote branch

**Review by ilyakaznacheev — APPROVED** [review]

**Pull request merged**